src/adapters/http/Problem.ts

```diff
@@ -6,6 +6,8 @@ export const fromErrors = (errors: ts.Errors): Problem => ({
   ...fromNumber(400),
   title: 'Bad Request',
   detail: PR.failure(errors),
+  timestamp: new Date(),
+  traceId: 'aTraceId',
```

Comment:
I think we should keep the format provided by PN
```suggestion
  traceId: 'Self=1-631b10db-2c571f3f79954187229939e5;Root=1-631b11db-39caf6636f4ed15618461d95;Parent=6e2ef00e703981ac;Sampled=1,
```

Reply:
Replaced with the correct value